exist-versioning-release.md

```diff
@@ -150,48 +231,72 @@ You will require a system with:
+                   <existdb.release.keystore.key.alias>YOUR-KEYSTORE-ALIAS</existdb.release.keystore.key.alias>
+                   <existdb.release.keystore.key.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.key.pass>
+               </properties>
+           </profile>
+        </profiles>
+        <activeProfiles>
+            <activeProfile>existdb-release-signing</activeProfile>
+        </activeProfiles>
     </settings>
+```
+
+1. Merge any outstanding PRs that have been reviewed and accepted for the milestone eXist-5.3.0.
+
+2. Create a new clone of the branch you want to release from into a new folder.
+   In the example below the `develop` branch is cloned into the folder `release`
+
+
+    ```bash
+    git clone \ 
+      -b develop \
+      git@github.com:eXist-db/exist.git \
+      release
+    ```
+    
+3. switch to that directory
+
+    ```bash
+    cd release
     ```
 
-2. You will need your GPG Key and Java KeyStore credentials for signing the release artifacts in the `<activeProfiles`> section on your machine in your local `~/.m2/settings.xml` file, e.g.:
-    ```xml
-    <profiles>
-   
-       <profile>
-           <id>existdb-release-signing</id>
-           <properties>
-               <existdb.release.key>ABC1234</existdb.release.key>
-               <existdb.release.public-keyfile>${user.home}/.gnupg/pubring.gpg</existdb.release.public-keyfile>
-               <existdb.release.private-keyfile>${user.home}/.gnupg/secring.gpg</existdb.release.private-keyfile>
-               <existdb.release.key.passphrase>your-password</existdb.release.key.passphrase>
-   
-               <existdb.release.keystore>${user.home}/your.store</existdb.release.keystore>
-               <existdb.release.keystore.pass>your-keystore-password</existdb.release.keystore.pass>
-               <existdb.release.keystore.key.alias>your-alias</existdb.release.keystore.key.alias>
-               <existdb.release.keystore.key.pass>your-key-password</existdb.release.keystore.key.pass>
-           </properties>
-       </profile>
-   
-    </profiles>
-
-
-    <activeProfiles>
-   
-           <activeProfile>existdb-release-signing</activeProfile>
-   
-    </activeProfiles>
+    You can verify that origin now points to the main repository `git@github.com:eXist-db/exist.git`
+    
+    ```bash
+    git remote -v
     ```
 
-3.  Merge any outstanding PRs that have been reviewed and accepted for the milestone eXist-5.3.0.
 
-4.  Make sure that you have the HEAD of `origin/develop` (or `upstream` if you are on a fork).
 
-5.  Prepare the release, if you wish you can do a dry-run first by specifiying `-DdryRun=true`:
-    ```
-    $ mvn -Ddocker=true -Dmac-signing=true -Dizpack-signing=true -Darguments="-Ddocker=true -Dmac-signing=true -Dizpack-signing=true" release:prepare
+4.  Prepare the release
+    **NOTE:** If this is the first time you prepare a release or you made changes to the process you should do a dry-run first by setting `-DdryRun=true`
```

Comment:
This should be reworded.

The release process is complex, you SHOULD ALWAYS do a dry-run first, regardless of how many times you have done this before. This saves having to undo bad release processes that stop 1/2 way through because perhaps you upgraded/removed changed GPG or some other external dependency.

Reply:
It was "if you wish you can do a dry-run first", you know. But it is okay to change that to a very rigid "do a dry-run first".